src/cli/cli.cpp

```diff
@@ -237,10 +237,29 @@ template <> otError Interpreter::Process<Cmd("version")>(Arg aArgs[])
 {
     otError error = OT_ERROR_NONE;
 
+    /**
+     * @cli version
+     * @code
+     * version
+     * OPENTHREAD/gf4f2f04; Jul 1 2016 17:00:09
+     * DONE
+     * @endcode
+     * @par api_copy
+     * #otGetVersionString
+     */
     if (aArgs[0].IsEmpty())
     {
         OutputLine("%s", otGetVersionString());
     }
+
+    /**
+     * @cli version api
+     * @code
+     * version api
+     * 28
+     * DONE
+     * @endcode
+     */
```

Comment:
Actually, since there's no API to link, you'll need to add a manual description here (after a `@par` tag).

Just use what's here: https://github.com/openthread/openthread/blob/main/src/cli/README.md#version-api

Reply:
done, thx